src/core/terms.ml

```diff
@@ -377,7 +377,7 @@ let _Impl : tbox -> tbox -> tbox =
 let _Abst : tbox -> tbinder Bindlib.box -> tbox =
   Bindlib.box_apply2 (fun a t -> Abst(a,t))
 
-(** [_Meta m ar] lifts the metavariable [m] to the {!type:tbox} type given its
+(** [_Meta u ar] lifts the metavariable [u] to the {!type:tbox} type given its
```

Comment:
Please replace u by m in the code instead.

Reply:
I propose to replace ar by ts as well.

Reply:
Done